decidim-proposals/lib/tasks/decidim_metrics_tasks.rake

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "decidim/metric_entity"
+
+namespace :decidim do
+  namespace :metrics do
+    # ProposalsMetric ------
+    #
+    # Execute ProposalsMetric's calculation method for today or a given date
+    task :proposals_metric, [:day] => :environment do |_task, args|
+      metric = Decidim::Proposals::Metrics::ProposalsMetricManage.for(args.day)
+      next unless metric.valid?
+      Decidim::Organization.all.each do |organization|
```

Comment:
@decidim/lot-core I'm generating so much duplicated code structure with this rake tasks. Do you know a way to *correct* this. taking care that each `metric` need to use a different manager